crates/re_web_viewer_server/src/lib.rs

```diff
@@ -12,6 +12,8 @@ use std::task::{Context, Poll};
 use futures_util::future;
 use hyper::{server::conn::AddrIncoming, service::Service, Body, Request, Response};
 
+pub const DEFAULT_WEB_VIEWER_PORT: u16 = 9090;
```

Comment:
```suggestion
pub const DEFAULT_WEB_VIEWER_SERVER_PORT: u16 = 9090;
```

Reply:
let's be annoyingly explicit, since web viewers also (often) connect to a WebSocket server…